### DISCUSSION TITLE
Question about rendering architecture (is angle used?)

### DISCUSSION THREAD

**Discussion by davidgiga1993** [MonoGame/MonoGame#8490] [iOS / iPadOS]
I looked a bit around but didn't find any specific documentation about it:
How does monogame render on iOS/MacOS? Does it use ANGLE given that openGL is deprecated in the Apple ecosystem?

**Comment by dellis1972** [comment]
Apple still uses OpenGL, even though it is deprecated it is still available to us. 
At some point in the future we will be replacing it with Vulkan or perhaps a Metal implementation.
